pkgs/by-name/it/itk_4_13/package.nix

```diff
@@ -0,0 +1,157 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  expat,
+  fftw,
+  fftwFloat,
+  hdf5-cpp,
+  libjpeg,
+  libtiff,
+  libpng,
+  libuuid,
+  xz,
+  vtk,
+  zlib
+}:
+  # this ITK version is old and is only required for OTB package
+  # https://gitlab.orfeo-toolbox.org/orfeotoolbox/otb/-/blob/develop/SuperBuild/CMake/External_itk.cmake?ref_type=heads#L149
+  stdenv.mkDerivation (finalAttrs: {
+    pname = "itk";
+    version = "4.13.3";
+
+    src = fetchFromGitHub {
+      owner = "InsightSoftwareConsortium";
+      repo = "ITK";
+      rev = "v${finalAttrs.version}";
+      hash = "sha256-lcoJ+H+nVlvleBqbmupu+yg+4iZQ4mTs9pt1mQac+xg=";
+    };
+
+    # https://gitlab.orfeo-toolbox.org/orfeotoolbox/otb/-/tree/develop/SuperBuild/patches/ITK?ref_type=heads
+    patches = [
+      ./itk-1-fftw-all.diff
+      ./itk-2-itktestlib-all.diff
+      ./itk-3-remove-gcc-version-debian-medteam-all.diff
+    ];
+
+    # https://gitlab.orfeo-toolbox.org/orfeotoolbox/otb/-/blob/develop/SuperBuild/CMake/External_itk.cmake?ref_type=heads
+    cmakeFlags = [
+      "-DBUILD_TESTING=OFF"
+      "-DBUILD_EXAMPLES=OFF"
+      "-DBUILD_SHARED_LIBS=ON"
+      "-DITK_BUILD_DEFAULT_MODULES=OFF"
+      "-DITKGroup_Core=OFF"
+      "-DITK_FORBID_DOWNLOADS=ON"
+      "-DITK_USE_SYSTEM_LIBRARIES=ON" # finds common libraries e.g. hdf5, libpng, libtiff, libjpeg, zlib etc
+#      "-DITK_USE_KWSTYLE=OFF"
+#      "-DITK_USE_SYSTEM_EIGEN=OFF"
+#      "-DITK_USE_SYSTEM_EXPAT=ON"
+#      "-DITK_USE_SYSTEM_ZLIB=ON"
+#      "-DITK_USE_SYSTEM_TIFF=ON"
+#      "-DITK_USE_SYSTEM_PNG=ON"
```

Comment:
You may want to remove this unused line.

Reply:
Thanks @jackyliu16 ! 

I have removed all the unused lines.